ci/build_test_OnCommitWin.groovy

```diff
@@ -6,6 +6,51 @@ pipeline {
         label 'win_ovms'
     }
     stages {
+        stage('Configure') {
+            steps {
+                script
+                {
+                    println "BUILD CAUSE: ${currentBuild.getBuildCauses()}"
+                    // BRANCH INDEXING BUILD CAUSE: [[_class:jenkins.branch.BranchIndexingCause, shortDescription:Branch indexing]]
+                    def isTriggeredByIndexing = currentBuild.getBuildCauses('jenkins.branch.BranchIndexingCause').size()
+                    // ON COMMIT TRIGGER BUILD CAUSE: [[_class:org.jenkinsci.plugins.workflow.support.steps.build.BuildUpstreamCause, 
+                    def isTriggeredByOnCommit = currentBuild.getBuildCauses('org.jenkinsci.plugins.workflow.support.steps.build.BuildUpstreamCause').size()  
+                    def isTriggeredByUser = currentBuild.getBuildCauses('hudson.model.Cause$UserIdCause').size()  
+                    def isTriggeredByTimer = currentBuild.getBuildCauses('hudson.triggers.TimerTrigger$TimerTriggerCause').size()
+
+                    if (isTriggeredByIndexing) {
+                        if (currentBuild.getNumber() == 1) {
```

Comment:
why first build in the indexing is to be executed?
Shouldn't we allow triggering by the user? 
If the build is triggered by other build, is it reported as user or indexing trigger?